frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/services/elements/ADS.js

```diff
@@ -47,5 +47,16 @@ export const ADSServices= {
       method: "POST",
       params: {},
       auth: true,
-  })
+  }),
+  estimateCreate:(details, tenantId) =>
```

Comment:
Same here

Reply:
this is for sending request to create a demand estimation in the ADS